components/cstudio-forms/controls/checkbox-group.js

```diff
@@ -212,7 +212,7 @@ YAHOO.extend(CStudioForms.Controls.CheckBoxGroup, CStudioForms.CStudioFormField,
                     groupEl.appendChild(rowEl);
                 }
                 _self.value = newValue;
-                _self.form.updateModel(_self.id, _self.getValue());
+                //_self.form.updateModel(_self.id, _self.getValue());
 
```

Comment:
Yes, this change was tested by Uber, He asked for the change, so I asked him to check it.
The problem was that this function was called in the init function, so it was changing the correct value to empty value. 
I'm going to remove it.